lms/templates/courseware/progress_graph.js

```diff
@@ -143,9 +269,23 @@ $(function () {
     
     %if show_grade_breakdown:
       var o = plot.pointOffset({x: ${overviewBarX} , y: ${totalScore}});
-      $grade_detail_graph.append('<div style="position:absolute;left:' + (o.left - 12) + 'px;top:' + (o.top - 20) + 'px">${"{totalscore:.0%}".format(totalscore=totalScore)}</div>');
+
+      edx.HtmlUtils.append(
+          $grade_detail_graph,
+          edx.HtmlUtils.joinHtml(
+              // safe-lint: disable=javascript-concat-html
+              edx.HtmlUtils.HTML('<div class="overallGrade" style="position:absolute;left:' + (o.left - 12) + 'px;top:' + (o.top - 20) + 'px">'),
+              edx.HtmlUtils.HTML('<span class=sr>'),
+              gettext('Overall Score'),
+              edx.HtmlUtils.HTML('</span>'),
```

Comment:
I added a br element within the span. Let me know if it needs to be outside of the span.